src/Controller/GameController.tsx

```diff
@@ -1,5 +1,8 @@
 import { LEVEL_UP_ROW_COUNT, MAX_LEVEL } from '../utils/Constants';
 
+const GAME_STATUSES = ['initial', 'pause', 'playing', 'gameOver'];
+export type GameStatus = (typeof GAME_STATUSES)[number];
```

Comment:
GAME_STATUSES is not used at all. Simplify:
```suggestion
export type GameStatus = 'initial' | 'pause' | 'playing' | 'gameOver';
```